FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -9,6 +9,8 @@
 import sys
```

Comment:
I think this is a good idea. So, If you can make a base class, let say BaseCLI, and place it in to DIRAC.Core.Base package and then inherit it in CSCLI.py to check and test, then afterwards we will be able to update other CLI clients accordingly.